### DISCUSSION TITLE
What are the Celery Flower Configuration Variables Good for?

### DISCUSSION THREAD

**Discussion by hbc-acai** [apache/airflow#33447] [Q&A] [answered]
Hello community, 

Just a general question about flower. I am wondering why these variables are in the celery config:

```
[celery]
flower_host = 0.0.0.0
flower_url_prefix = 
flower_port = 5555
flower_basic_auth = 
```
Are these variables used for celery workers to communicate with the flower service? I have my celery workers and flower running on different hosts now. Without setting these variables, flower seems know all the celery workers already. 

Thank you!

**Comment by Taragolis** [comment]
This is default configurations for [`airflow celery flower`](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#flower) command.

**Comment by andresitorusz** [comment, marked as answer]
Flower is a web-based tool for monitoring and administrating Celery clusters. This topic describes how to configure Airflow to secure your flower instance.

This is an optional component that is disabled by default in Community deployments and you need to configure it on your own if you want to use it.

Read more here -> https://airflow.apache.org/docs/apache-airflow/stable/security/flower.html#flower
